test/unit/sayt.ts

```diff
@@ -173,9 +170,9 @@ suite('Sayt', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveAlias })
   describe('setInactive()', () => {
     it('should call unregisterClickAwayHandler()', () => {
       const unregisterClickAwayHandler = sayt.unregisterClickAwayHandler = spy();
-      stub(Selectors, 'query');
+      sayt.select = spy(() => undefined);
```

Comment:
the default behaviour of a spy is to return undefined, so this can just be `sayt.select = spy()`